alphagov/paas-cf-apps-statsd
# Update import paths after repo rename

## What

As the repository name as a separate step the import paths still
contained the old repository name for each package import, compilation
failed.

This commit updates the imports to use the new repo name.

## How to test

Clone this repository and switch to the `fix-import-paths` branch.
Run the tests and verify all pass.

## Who can review?

Not @LeePorte
